src/overcooked_ai_py/mdp/overcooked_env.py

```diff
@@ -664,6 +665,136 @@ def proportion_stuck_time(trajectories, agent_idx, stuck_time=3):
         return stuck_matrix
 
 
+class OvercookedEnv_PettingZoo(gymnasium.Wrapper):
+    """
+    Create environment for gymnasium for pettingZoo
+    """
+
+    def step(self, joint_action):
+        obs, reward, done, info = super().step(joint_action)
+        # from the doc, this is true only if a pre set goal is achieved, which we don't have in this environment
+        # the only termination condition is the preset max steps in the environment
+        terminated = False
+        truncated = done
```

Comment:
Is the done signal sent only when a time limit is reached? Usually I’d expect terminated to be set to done and truncated to be determined by a time limit

Reply:
Yeah that's correct. The goal for agents in this environment is to obtain as much reward as possible under a fixed amount of time, so there is no early-termination condition and there is only the time limit. 

Just one other thing, this class (OvercookedEnv_PettingZoo) is not actually used and was a leftover when I was playing around with PettingZoo. The actual class is the one below (OvercookedEnv_Pet).